config/src/main/java/no/sysco/middleware/kafka/clients/interceptor/config/Utils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package no.sysco.middleware.kafka.clients.interceptor.config;
+
+import java.util.Collections;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.errors.TopicExistsException;
+
+/**
+ *
+ * @author SYSCO Middleware
+ */
+public class Utils {

```

Comment:
Not sure if any other class will reuse these methods like to create a Utils class. We could just add them to the class where it is used.

Reply:
fixed